### PR TITLE
Ref Task 6252: Create selected genre screen ui

### PR DESCRIPTION
**Pull request by trieuthan92pro**
Link redmine: https://edu-redmine.framgia.vn/issues/6252
screenshot_2019-02-19-15-22-41-487_com example soundcloud

### CODE REVIEW

**Comment by trieuthan92pro** [comment]
Ready

File: app/src/main/res/layout/activity_selected_genre.xml [hunk 1]
```diff
@@ -7,4 +7,64 @@
     android:layout_height="match_parent"
     tools:context=".selected_genre_detail.SelectedGenreActivity">
 
+    <android.support.v7.widget.Toolbar
+        android:id="@+id/toolbar2"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_selected_genre.xml]:
```suggestion
        android:id="@+id/tool_bar_second"
```

File: app/src/main/res/drawable/ic_file_download_24dp.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#850885"
+    android:viewportHeight="24.0" android:viewportWidth="24.0"
+    android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#FF000000" android:pathData="M19,9h-4V3H9v6H5l7,7 7,-7zM5,18v2h14v-2H5z"/>
+</vector>
```
Comment (nvquangth) [inline app/src/main/res/drawable/ic_file_download_24dp.xml:5]: Format code

File: app/src/main/res/drawable/ic_search_24dp.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#616161"
+    android:viewportHeight="24.0" android:viewportWidth="24.0"
+    android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#FF000000" android:pathData="M15.5,14h-0.79l-0.28,-0.27C15.41,12.59 16,11.11 16,9.5 16,5.91 13.09,3 9.5,3S3,5.91 3,9.5 5.91,16 9.5,16c1.61,0 3.09,-0.59 4.23,-1.57l0.27,0.28v0.79l5,4.99L20.49,19l-4.99,-5zM9.5,14C7.01,14 5,11.99 5,9.5S7.01,5 9.5,5 14,7.01 14,9.5 11.99,14 9.5,14z"/>
+</vector>
```
Comment (nvquangth) [inline app/src/main/res/drawable/ic_search_24dp.xml:5]: Format code

File: app/src/main/res/layout/song_item.xml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_16"
+        android:layout_marginEnd="@dimen/margin_16"
+        android:layout_marginRight="@dimen/margin_16"
+        android:layout_marginBottom="@dimen/margin_16"
+        android:src="@drawable/ic_more_fill_vertical"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+         />
```
Comment (nvquangth) [inline app/src/main/res/layout/song_item.xml:62]: Format code

File: app/src/main/res/layout/song_item.xml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_16"
+        android:layout_marginEnd="@dimen/margin_24"
+        android:layout_marginRight="@dimen/margin_24"
+        android:layout_marginBottom="@dimen/margin_16"
+        android:src="@drawable/ic_file_download_24dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/image_song_item_option"
+        app:layout_constraintTop_toTopOf="parent"
+         />
```
Comment (nvquangth) [inline app/src/main/res/layout/song_item.xml:76]: Format code

File: app/src/main/res/menu/song_item_menu.xml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    app:showAsAction="always">
+    <item
+        android:id="@+id/action_add_contact"
+        android:icon="@drawable/ic_file_download_24dp"
+        android:title="Download"
```
Comment (nvquangth) [inline app/src/main/res/menu/song_item_menu.xml:8]: Hardcode

**Comment by trieuthan92pro** [comment]
Ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**